spec/spec.md

```diff
@@ -186,30 +186,34 @@ The following process defines how a DID-Relative URL for a Decentralized Web Nod
 **DID-Relative URL example for passing multiple messages:**
 
 ::: note
-For example purposes, the `queries` parameter value below is neither JSON stringified nor Base64Url encoded, but should be when using Decentralized Web Node URLs in practice (see the [DID-relative URL Composition](#composition) instructions above).
+For example purposes, the `message` parameter value below is neither JSON stringified nor Base64Url encoded, but should be when using Decentralized Web Node URLs in practice (see the [DID-relative URL Composition](#composition) instructions above).
 :::
 
 ```json
-did:example:123?service=DecentralizedWebNode&queries=[{ "interface": "Records", "method": "Query", "schema": "https://schema.org/SocialMediaPosting" }]
+did:example:alice?service=DecentralizedWebNode&message={ "interface": "Records", "method": "Query", "schema": "https://schema.org/SocialMediaPosting" }
 ```
 
 ```json
-did:example:123?service=DecentralizedWebNode&queries=W3sgTUVTU0FHRV8xIH0sIHsgTUVTU0FHRV9OIH1d...
+did:example:alice?service=DecentralizedWebNode&message=W3sgTUVTU0FHRV8xIH0sIHsgTUVTU0FHRV9OIH1d...
 ```
 
 **Resolve DID to locate its Decentralized Web Node URIs:**
 
-`did:example:123` -->  resolve to Decentralized Web Node endpoint(s)  -->  `https://dwn.example.com/`
+`did:example:alice --> resolve DWN endpoint(s) --> `https://dwn.example.com`
```

Comment:
See my point above. We should change the example to have `https://dwn.example.com` as the first listed endpoint, given the service resolution preferences described in https://identity.foundation/decentralized-web-node/spec/#resolution, #3.